vite.config.js

```diff
@@ -42,4 +42,3 @@ export default defineConfig({
         },
     },
 });
```

Comment:
Add a blank line at the end of the file

Reply:
![image](https://user-images.githubusercontent.com/62728106/163028720-90efcdf0-c364-4f4e-af1b-a158c894c217.png)
fixed